src/patternfly/components/Drawer/drawer-action.hbs

```diff
@@ -0,0 +1,8 @@
+<{{#if drawer-action--type}}{{{drawer-action--type}}}{{else}}div{{/if}} class="pf-c-drawer__action{{#if drawer-action--modifier}} {{drawer-action--modifier}}{{/if}}"
+  {{#if drawer-action--attribute}}
+    {{{drawer-action--attribute}}}
+  {{/if}}>
+  {{#> button button--modifier="pf-m-plain" button--id=(concat drawer--id '-close-button') button--attribute=(concat 'id="'  drawer--id '-close-button" aria-labelledby="' drawer--id '-close-button ' drawer--id '-label" aria-label="Close drawer panel"')}}
```

Comment:
should we only use `aria-labelledby` if the `-label` element exists? That element may not always exist, since the `-label` isn't an element of the drawer component. (like it isn't in any of these examples https://patternfly-next-pr-2645.surge.sh/documentation/core/beta/drawer)